Lib/test/test_unittest/test_result.py

```diff
@@ -527,9 +524,9 @@ def testGetDescriptionWithOneLineDocstring(self):
         result = unittest.TextTestResult(None, True, 1)
         self.assertEqual(
                 result.getDescription(self),
-               ('testGetDescriptionWithOneLineDocstring '
-                '(' + __name__ + '.Test_TextTestResult.testGetDescriptionWithOneLineDocstring)\n'
-                'Tests getDescription() for a method with a docstring.'))
+               ('Tests getDescription() for a method with a docstring. '
+                '(' + __name__ + '.Test_TextTestResult.testGetDescriptionWithOneLineDocstring)'
```

Comment:
I will not mark every occurrence, but you can use f-strings as well when you change the code (don't put f-strings where you don't change the code).

It's hard to see on GH but try to fit within 79 characters per line (maybe you'll need temporary variables).

Reply:
I fixed the f-strings.

Not sure about the line length though. It seems to be quite commonly violated in tests and no automatic check complains either. Some lines I modified exceeded the limit before as well.

Reply:
I think you can fix your own lines if you want, and leave others untouched. But yes, in general PEP 8 is not really respected in tests. If 80 is too small, just try not to exceed 95-100 (otherwise, it will really be outside of most of the screens).

Reply:
OK, I cut some of the lines down.